src/components/modules/nutrition/daily-intake/DailyIntakeFoods.vue

```diff
@@ -6,8 +6,8 @@
                     <v-list-item-content>
                         <v-list-item-title>
                             <div class="title-container">
-                                <span>{{ food.name }}</span>
-                                <span>{{ food.numberOfPortions * food.portion | cutDigits }} g</span>
+                                <span class="constrained">{{ food.name }}</span>
+                                <span>{{ food.numberOfPortions * food.portion | cutDigits }} {{food.portionUnit ? food.portionUnit : 'g'}}</span>
```

Comment:
I would move content to new line as this line is quite long and harder to read:
```
<span>
  {{ food.numberOfPortions * food.portion | cutDigits }} {{food.portionUnit ? food.portionUnit : 'g'}}
</span>
```

Reply:
Done